Add info on metric correlations support (#20519)
DataDog/documentation: content/en/dashboards/correlations/_index.md

* Left click on any graph and select **Find correlated metrics**.
* From a full-screen graph, click the **Correlations** tab.

{{< img src="dashboards/correlations/find_correlated_metrics.png" alt="Dashboard graph menu option find correlated metrics" style="width:80%;">}}

{{< img src="dashboards/correlations/correlations_tab.png" alt="Dashboard search" style="width:80%;">}}

> Correlations *tries* to automatically detect the area of interest (anomalous behavior) for your metric. If the area of interest is not selected automatically or needs adjustment, you can manually draw the area of interest from the [edit search](#edit) option. Datadog searches for other metrics that exhibited anomalous behavior in a time aligned with the area of interest.

Reviewer: ```suggestion
Correlations *tries* to automatically detect the area of interest (anomalous behavior) for your metric. If the area of interest is not selected automatically or needs adjustment, you can manually draw the area of interest from the [edit search](#edit) option. Datadog searches for other metrics that exhibit anomalous behavior at times matching the area of interest.
```
